lib/js-preprocessor/index.js

```diff
@@ -8,7 +8,7 @@ var funnel = require('broccoli-funnel');
 module.exports = function(tree, options) {
   var generated = new GenerateComponents(tree, options);
 
-  var merged = mergeTrees([tree, generated]);
+  var merged = mergeTrees([tree, generated], { overwrite: true });
```

Comment:
We should try and not use this. If someone writes a base component manually, we don't want it to be overridden by the auto generated component